.gitignore

```diff
@@ -3,3 +3,4 @@ dist
 node_modules
 .vscode-test
 *.vsix
+.vscode-test-web
```

Comment:
Can we collapse this into the above and have a `.vscode-test*` gitignore rule?

Reply:
Yes